pom.xml

```diff
@@ -64,7 +64,7 @@
     <properties>
         <revision>2.41</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.176.4</jenkins.version>
+        <jenkins.version>2.222.3</jenkins.version>
```

Comment:
I would prefer to still support older LTS lines for a widely-used plugin like this. Is it possible to use reflection and drop the `@Override` annotation on `getRequiredGlobalConfigPagePermission` so that we can keep the core dependency as-is while still supporting the new behavior for anyone running the latest LTS?

Reply:
Sure! Done.